### PR TITLE
API-13275/  docserver  update docker compose.yml volumes

### PR DESCRIPTION
**Pull request by mheusgit**
- Removed blanket volume mounting of working directory to focus on folders that need it
- Updated dockerignore

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -12,8 +12,12 @@ services:
       - 8080:8080
       - 9229:9229
     volumes:
-      - .:/home/node
-      - nm:/home/node/node_modules
+      - ./src:/home/node/src
+      - ./test:/home/node/test
+      - ./dev:/home/node/dev
```
Comment (dsteeve) [inline docker-compose.yml:17]: this one is covered in the docker-compose sample-oas-service

**Review by dsteeve — COMMENTED** [review]
looks good -- one minor change requested
let's isolate this change to its own MR

File: docker-compose.yml (hunk 1 above)
Comment (mheusgit) [reply docker-compose.yml]: Line was removed

**Review by dsteeve — APPROVED** [review]
did an in-person code review and requested change is in.  looks good!

**Pull request merged**